frame/fast-unstake/src/lib.rs

```diff
@@ -138,6 +135,14 @@ pub mod pallet {
 
 		/// The weight information of this pallet.
 		type WeightInfo: WeightInfo;
+
+		/// Maximum value for `ErasToCheckPerBlock`. This should be as close as possible, but more
+		/// than the actual value, in order to have accurate benchmarks.
+		type MaxErasToCheckPerBlock: Get<u32>;
```

Comment:
Its quite sad that we still cannot express relations between config types…  
Both options are a bit ugly and maybe not as widely applicable as it first looks.

```suggestion
		type MaxErasToCheckPerBlock: Get<u32, Min=Self::ErasToCheckPerBlock>;
```
of 
```suggestion
		#[constraint(self >= Self::ErasToCheckPerBlock)]
		type MaxErasToCheckPerBlock: Get<u32>;
```

Reply:
😢